src/components/UI/common/Footer.tsx

```diff
@@ -0,0 +1,105 @@
+import { Box, Center, Divider, Flex, Link, Stack } from '@chakra-ui/react';
+import Image from 'next/image';
+import { FC } from 'react';
+import efLogoSVG from '../../../public/images/ef-logo.svg';
+import githubLogoSVG from '../../../public/images/github-logo.svg';
+import twitterLogoSVG from '../../../public/images/twitter-logo.svg';
+import { ImportantText, PageText } from '../text';
+
+export const Footer: FC = () => {
+  return (
+    <Flex id='footer' bgGradient='linear(to-b, #E07861 10%, #E06361 100%)' d='column'>
+      <Center px={5} py={8}>
+        <Image src={efLogoSVG} alt='Ethereum Foundation logo' height={64} width={200} />
+      </Center>
+
+      <Center px={5}>
+        <Divider borderColor='#e5e5e5' opacity={1} />
```

Comment:
Same here - should this color be a variable?

Reply:
will move the colors to variables 👌